store/migration/dev/20220714000000__backup_retention.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE backup_setting ADD retention_period_ts INTEGER NOT NULL;
```

Comment:
Let's separate schema change and code change.

https://github.com/bytebase/bytebase/blob/main/docs/schema-update-guide.md#development

Reply:
SG.

Reply:
Created another PR for schema change.